### DISCUSSION TITLE
[Bug]: Heltec V3 boot loop with 2.4.2 and 2.5.0

### DISCUSSION THREAD

**Discussion by soragan** [meshtastic/firmware#4548] [Troubleshooting]
### Category

Hardware Compatibility

### Hardware

Heltec V3

### Firmware Version

2.4.2+

### Description

Using web flasher to update firmware and device went into boot loop on 2.4.2 and 2.5.0. Tested on 2.4.1 and working fine

### Relevant log output

```Shell
load:0x403c9700,len:0xbd8
load:0x403cc700,len:0x2a0c
entry 0x403c98d0
E (339) esp_core_dump_flash: No core dump partition found!
E (339) esp_core_dump_flash: No core dump partition found!
��@INFO  | ??:??:?? 0
```

**Comment by thebentern** [comment]
Try a full erase and install. V3s are working fine here.

**Comment by soragan** [comment]
downgraded to 2.4.1. Will try full install in couple of days
